deploy/kubernetes/elastic-agent-managed/elastic-agent-managed-daemonset.yaml

```diff
@@ -62,6 +62,12 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: KUBERNETES_NODE_NAME
```

Comment:
I think that ${env.NODE_NAME} is available for you to use. Can you check if you can use it and not need this block?

Reply:
`KUBERNETES_NODE_NAME` is a commonly used environment variable within the Elastic org: https://github.com/search?type=code&q=org%3Aelastic+KUBERNETES_NODE_NAME

And similar to other k8 related environment variables, like `KUBERNETES_SERVICE_PORT_HTTPS` or `KUBERNETES_SERVICE_HOST`, I think it is good to be more explicit and keep the `KUBERNETES_` prefix.
Would it be ok to comment it out for the begining?

Reply:
I believe  having  both NODE_NAME and KUBERNETES_NODE_NAME in the same manifest does not make sense and can look very weird to customers. Besides that KUBERNETES_NODE_NAME is commonly used, is there any other reason that you added it?

Reply:
`KUBERNETES_NODE_NAME` is an essential part of detecting and working in the k8 environment for this integration. Without it, the integration will not work.